src/components/App/app.test.tsx

```diff
@@ -0,0 +1,78 @@
+// app.test.tsx
```

Comment:
Test fails. So file to remove. To be done later

Reply:
I've disable it to not forget it later